app/helpers/layout_helper.rb

```diff
@@ -45,8 +45,8 @@ def translation_missing_warnings
     end
   end
 
-  def include_base_css_framework
-    stylesheet_link_tag('bootstrap-complete')
+  def include_color_theme view ="desktop"
```

Comment:
Use def with parentheses when there are parameters.<br>Surrounding space detected in default value assignment.